pkg/apply/task/inv_set_task_test.go

```diff
@@ -164,18 +166,25 @@ func TestInvSetTask(t *testing.T) {
 
 	for name, tc := range tests {
 		t.Run(name, func(t *testing.T) {
-			client := inventory.NewFakeClient(object.ObjMetadataSet{})
+			// client := inventory.NewFakeInventoryClient(object.ObjMetadataSet{})
+			client := &inventory.InMemoryClient{}
```

Comment:
Does it make sense to replace the `inventory.NewFakeClient` with the `inventory.InMemoryClient` before changing the client interface? Or would that just be wasted effort?